src/main.c

```diff
@@ -255,7 +228,16 @@ static void activate(GtkApplication *app, gpointer *user_data) {
     gtk_container_add(GTK_CONTAINER(main_window), GTK_WIDGET(main_grid));
 
     gtk_widget_show_all(main_window);
-    g_timeout_add(10, on_timeout, app_data);
```

Comment:
I don't know why this was done with a timeout before. As far as I can tell, connecting to ALSA mixer immediately works fine.